src/chrome/background/background.ts

```diff
@@ -114,3 +116,31 @@ chrome.runtime.onStartup.addListener(() => {
 })
 
 createOffscreen()
+
+chrome.contextMenus?.removeAll(() => {
+  if (isDevMode) {
+    chrome.contextMenus.create({
+      id: 'radix-dev-tools',
+      title: 'Radix Dev Tools',
+      contexts: ['all'],
+    })
+  }
+
+  chrome.contextMenus.create({
+    id: 'radix-ce-logs',
+    title: 'Export Logs',
+    contexts: ['all'],
+  })
+})
+
+chrome.contextMenus.onClicked.addListener(async (data) => {
+  switch (data.menuItemId) {
+    case 'radix-dev-tools':
+      openRadixDevToolsPage()
+      break
+    case 'radix-ce-logs':
+      const logs = JSON.stringify(await chrome.storage.session.get(null))
```

Comment:
we should store logs under a key instead of the whole session storage

Reply:
We need to build a queue with logs to save. For each log message that comes to background script we need to first read all logs from storage, append current message and save all logs(which are async operations). Considering we may have many logs coming in short period of time does it sound ok? We're going keep reading + saving to `chrome.storage.sync` almost all the time.

I'm thinking if some structured data storage (IndexedDb, directly from offscreen) would be worth investigating for this use case🤔